universal-login-wallet/src/index.tsx

```diff
@@ -10,7 +10,7 @@ import {CustomBrowserRouter} from './ui/react/CustomBrowserRouter';
 const config = getConfig();
 
 const services = createServices(config);
-services.tokenService.start();
+services.tokensDetailsStore.aggregateTokensDetails();
```

Comment:
We should aggregate all start methods into one `start()` methods of `service` object.